tests/conftest.py

```diff
@@ -1217,7 +1217,7 @@ def get_completeness_level_metadata(request):
     # if completeness_level is not set or an unknown completeness_level is set
     # return "thorough" to run all test set
     if not completeness_level or completeness_level not in ["debug", "basic", "confident", "thorough"]:
-        return "thorough"
+        return "basic"
```

Comment:
How about using default value `debug`?

Reply:
Is 'basic' level implemented in other test modules already?

Reply:
In most of test modules, 'basic' level was used as default, I think we can discuss and check all modules later, if there is no risk, will create another PR to modify all modules to use 'debug' as default level

Reply:
> Is 'basic' level implemented in other test modules already?

Yes

Reply:
> > How about using default value `debug`?
> 
> In most of test modules, 'basic' level was used as default, I will check all modules later, if there is no risk, will create another PR to modify all modules to use 'debug' as default level

I don't get it. I mean, this is the global default. If a test script has it's own default completness level, use its own. Otherwise, fallback to use the global default. `debug` level requires least run time, right?

For now, we can make global default `debug`. For test scripts, use separate PRs to update their completness level when necessary.

Reply:
> > > How about using default value `debug`?
> > 
> > 
> > In most of test modules, 'basic' level was used as default, I will check all modules later, if there is no risk, will create another PR to modify all modules to use 'debug' as default level
> 
> I don't get it. I mean, this is the global default. If a test script has it's own default completness level, use its own. Otherwise, fallback to use the global default. `debug` level requires least run time, right?
> 
> For now, we can make global default `debug`. For test scripts, use separate PRs to update their completness level when necessary.

got it, will change